src/main/groovy/org/labkey/gradle/util/BuildUtils.groovy

```diff
@@ -353,103 +353,70 @@ class BuildUtils
 
     static String getVersionNumber(Project project)
     {
+        String version = project.labkeyVersion
+
         if (project.hasProperty("versioning"))
         {
             String branch = project.versioning.info.branchId
-            // When a git module is on the "master" branch in git, this corresponds to the alpha branch
-            // at the root and we want to use that version for consistency with the SVN modules
-            if (branch.equalsIgnoreCase("master"))
-                return project.rootProject.version
-            else if (["trunk", "develop", "none"].contains(branch) || branch.toLowerCase().matches("release.*-snapshot"))
-                return project.labkeyVersion
-            else
+            if (!["trunk", "develop", ""].contains(branch) && !branch.toLowerCase().matches("release.*-snapshot"))
```

Comment:
Are you sure "none" is not a possibility still?  We should probably also include "master" here.